browser.js

```diff
@@ -5,12 +5,29 @@ var format = require('quick-format-unescaped')
 module.exports = pino
 
 var _console = global.console || {}
+var stdSerializers = {
+  req: mock,
+  res: mock,
+  err: asErrValue
+}
 
 function pino (opts) {
   opts = opts || {}
   opts.browser = opts.browser || {}
   var proto = opts.browser.write || _console
   if (opts.browser.write) opts.browser.asObject = true
+  var serializers = opts.serializers || {}
+  var serialize = Array.isArray(opts.browser.serialize)
+    ? opts.browser.serialize.filter(function (k) {
+      return k !== '!stdSerializers.err'
+    })
+    : opts.browser.serialize === true ? Object.keys(serializers) : false
+  var stdErrSerialize = opts.browser.serialize
+
+  if (
+    Array.isArray(opts.browser.serialize) &&
+    opts.browser.serialize.indexOf('!stdSerializers.err') > -1
```

Comment:
Does `[].includes()` not exist in browsers?

Reply:
because it's browser-side code I'm going for lowest common denominator, includes is ES7 so it's definitely not going to be in IE11 down which I'll wager are still in use - however we could do a polyfill since `includes` is actually more performant (it exits once an element is discovered). I'm not sure if the extra code is worth it at this stage, would rather upgrade in the future when we know there will be no breakage.

Reply:
to elaborate further: the priority optimization with browser pino is payload optimization rather than execution speed, this is why I've stayed away from polyfills. 

You could argue that `Object.assign` is out baseline though, if all browsers that support `Object.assign` support `Array.prototype.includes` then it's safe to use and for legacy browsers it's on the user to polyfill if the need arises 

Still, from an opportunity cost perspective, for now I'm not sure this is worth the research to decide on the change (having said that, happy upon insistence to look into)

Reply:
I was seriously asking. I do so little front end stuff that I have no idea what is available there.

Reply:
on cursory look there's no IE support for includes, but there's also no support for Object.assign so...